backend/src/modules/student/student_entity.py

```diff
@@ -16,10 +16,37 @@ class StudentEntity(EntityBase):
     id: Mapped[int] = mapped_column(Integer, primary_key=True, index=True)
     first_name: Mapped[str] = mapped_column(String, nullable=False)
     last_name: Mapped[str] = mapped_column(String, nullable=False)
-    call_or_text_pref: Mapped[CallOrTextPref] = mapped_column(
-        Enum(CallOrTextPref), nullable=False
+    call_or_text_pref: Mapped["CallOrTextPref"] = mapped_column(
+        Enum("CallOrTextPref"), nullable=False
     )
     last_registered: Mapped[DateTime | None] = mapped_column(DateTime, nullable=True)
```

Comment:
This is my fault, had to rush to get these entities in time. This DateTime in  Mapped should be the `datetime` from the python `datetime` library. If you could make that change for me as you go through the others that would be great
```suggestion
    last_registered: Mapped[datetime | None] = mapped_column(DateTime, nullable=True)
```